dogpile/cache/backends/file.py

```diff
@@ -140,6 +140,13 @@ def release_write_lock(self):
     """
 
     def __init__(self, arguments):
+        super().__init__(
+            {
+                "pickler": pickle.dumps,
+                "unpickler": pickle.loads,
+                **arguments
```

Comment:
oh, you are using some flashy new python 3 syntax thing.    that syntax is a little icky since it is in my 2 minutes experience pretty hard to spot, but that's on them not you : )

Reply:
We can also use [ChainMap](https://docs.python.org/3/library/collections.html#collections.ChainMap), as I think it was created exactly for this purpose.